.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+#### Description
+
+<!--  The description should give a general overview of the goal of the PR.
+
+Individual commit message should highlight important changes that people may
+want to know about year later when sorting thorough the code.
+-->
+
+#### Checklist
+
+<!-- replace [ ] with [x] to select -->
+<!-- (strike not applicable items with ~~ around the text) -->
+
+- [ ] Code was modified
```

Comment:
```suggestion
- [ ] Code was modified for feature?
```
Without the addition this checkbox really sounds to me like meaningless checkbox ...